manifests/0000_90_kube-controller-manager-operator_03_servicemonitor.yaml

```diff
@@ -42,3 +42,21 @@ spec:
       for: 15m
       labels:
         severity: critical
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: kube-controller-manager-operator
+  namespace: openshift-kube-controller-manager-operator
+spec:
+  groups:
+  - name: cluster-version
+    rules:
+    - alert: PodDisruptionBudgetMisconfigured
```

Comment:
PodDisruptionBudgetPreventsDrain

Reply:
Actually..

PodDisruptionBudgetAtLimit = PDB expected == health: "The pod disruption budget is preventing further disruption to pods because it is at the minimum allowed level" - warning
PodsBelowDisruptionLimit = PDB expected < health: "The pod disruption budget is below the minimum number allowed pods" - critical